### PR TITLE
Feature/9 option menu worlds

### PR DESCRIPTION
**Pull request by aurelenc**

### CODE REVIEW

**Review by MisterPeModder — APPROVED** [review]

**Review by MisterPeModder — CHANGES_REQUESTED** [review]
The whole object-oriented architecture just to create a few worlds seems over-engineered to me.

**Comment by aurelenc** [comment]
> The whole object-oriented architecture just to create a few worlds seems over-engineered to me.

no worries

**Review by MisterPeModder — APPROVED** [review]

File: src/game/scenes/MainScene.hpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+#include "ecs/World.hpp"
+#include "game/scenes/AScene.hpp"
+
+namespace game
+{
+    class MainScene : public AScene {
+      public:
+        MainScene();
+    };
```
Comment (ZQUEMA) [inline src/game/scenes/MainScene.hpp:19]: Rename MainScene as MainMenuScene
This is more comprehensible

File: src/game/scenes/TestScene.cpp [hunk 2]
```diff
@@ -107,38 +111,40 @@ static void addTestWidgets(ecs::World &world)
     world.addSystem<game::systems::InputManager>();
 }
 
+static void loadTestWorld(ecs::World &world)
```
Comment (ZQUEMA) [inline src/game/scenes/TestScene.cpp:114]: Rename as loadTestScene

**Review by ZQUEMA — CHANGES_REQUESTED** [review]
Changes are requested

**Review by AndreasLrx — APPROVED** [review]
Fine for me when @ZQUEMA requests are resolved.

**Pull request merged**